incubator/java-microprofile/templates/default/src/main/java/dev/appsody/starter/HelloResource.java

```diff
@@ -0,0 +1,17 @@
+package dev.appsody.starter;
```

Comment:
Graham Charters and I discussed this when we first agreed on what to add. We decided not to add this one as this file will be deleted first. We just added something absolutely needed.

Reply:
I had a chat with Graham again this morning. According to our survey, developers don't want a lot of code to be generated as they have to delete. This file will be the first one to be deleted. I suggest not to add in. This stack is for production use. We need to keep it concise and useful. Therefore, I suggest not to add this file.